src/sentry/integrations/jira_server/client.py

```diff
@@ -100,6 +108,12 @@ class JiraServerClient(ApiClient):
     Client for making authenticated requests to JiraServer
     """
 
+    META_URL = '/rest/api/2/issue/createmeta'
```

Comment:
I feel like Jira Server would particularly benefit from a link to the api docs. I found finding the right flavor of jira server confusing. Even if it's just a link in the docstring of the client.

Reply:
:+1: